File: Bee-Crowd/Exercises_C++/1134.cpp
#include <iostream>
using namespace std;

int main() {

    int al,ga,di,count;
    al = 0;
    ga = 0;
    di = 0;
    count = 0;

    while (count != 4) {
        cin >> count;

        if (count == 1) {
            al += 1;
        }
        else if (count == 2) {
            ga += 1;
        }
        else if (count == 3) {
            di += 1;
        }
    }
    
    cout << "MUITO OBRIGADO" << endl;
    cout << "Alcool: " << al << endl;
    cout << "Gasolina: " << ga << endl;
    cout << "Diesel: " << di << endl;
}

File: Bee-Crowd/Exercises_Python/1134.py
numero = int(input())
alcool = []
gasolina = []
diesel = []

while numero != 4:
    if numero == 1:
        alcool.append(numero)
    elif numero == 2:
        gasolina.append(numero)
    elif numero == 3:
        diesel.append(numero)
    numero = int(input())

print('MUITO OBRIGADO')
print('Alcool:',len(alcool))
print('Gasolina:',len(gasolina))
print('Diesel:',len(diesel))